### PR TITLE
fix: Sidebar scrolling misclick issue on iOS

### PR DESCRIPTION
**Pull request by MartinSchoeler**
## Proposed changes (including videos or screenshots)

An outdated polyfill was causing misclick issues when scrolling the channels sidebar

## Further comments

[SUP-582]

[SUP-582]: https://rocketchat.atlassian.net/browse/SUP-582?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/meteor/client/polyfills/hoverTouchClick.ts [hunk 1]
```diff
@@ -1,17 +0,0 @@
-import * as domEvents from '../lib/utils/domEvents';
-import { isIOsDevice } from '../lib/utils/isIOsDevice';
```
Comment (yash-rajpal) [inline apps/meteor/client/polyfills/hoverTouchClick.ts:2]: The helper `isIosDevice` is not being used anywhere else in the project, we should remove that as well.

**Review by yash-rajpal — CHANGES_REQUESTED** [review]
LGTM other than the comment left.

File: apps/meteor/client/polyfills/hoverTouchClick.ts (hunk 1 above)
Comment (tassoevan) [reply apps/meteor/client/polyfills/hoverTouchClick.ts:2]: Even better! More deleted lines!

Comment (MartinSchoeler) [reply apps/meteor/client/polyfills/hoverTouchClick.ts:2]: Done!

**Review by yash-rajpal — APPROVED** [review]

**Comment by MartinSchoeler** [comment]
Putting this as draft since the issue seems more complex than initially estimated

**Review by dougfabris — CHANGES_REQUESTED** [review]
fix missing changeset

**Pull request merged**